packages/flutter/test/material/dropdown_test.dart

```diff
@@ -2949,4 +2949,57 @@ void main() {
     expect(scrollController.position.maxScrollExtent > 0, isTrue);
     expect(find.byType(Scrollbar), paints..rect());
   });
+
+  // Regression test for https://github.com/flutter/flutter/issues/76614
+  testWidgets('Do not crash if used in very short screen', (WidgetTester tester) async {
+    const String value = 'foo';
+    final UniqueKey itemKey = UniqueKey();
+    await tester.pumpWidget(
+      Center(
+        child: SizedBox(
+          height: 48.0 * 3 - 1.0, // The height of menu is 47.0.
```

Comment:
We may want to specify why we use 3x - 1.0 (I know from working on a previous PR that we need two items padding since the menu requires empty space surrounding the menu, but people reading just this test by itself wouldn't)